### ISSUE TITLE
Block must not be padded by blank lines.

### ISSUE THREAD

**Issue by kytwb** [eslint/eslint#2224]
On the first line of this snippet, column 80:

```javascript
  it('should throw an exception when cache configuration is wrong', function () {
    var c = {                                                                                                                                                             
      store: {
        put: function () {}                                                                                                                                               
      }
    }
```

**Comment by kytwb** [comment]
Ah, my bad, that was correct but the padding was on the end of the block.
Would be cool if it was referencing to the line where the padding match, and not the beginning of the block.

**Comment by lo1tuma** [comment]
Agreed, the rule always reports the start position of the block, but it should report the position of the extra/missing padding.

**Comment by nzakas** [comment]
@kytwb

**Comment by nzakas** [comment]
@kytwb please post the ESLint output you are seeing for this code.

**Comment by kytwb** [comment]
```
112  it('should throw an exception when cache configuration is wrong', function () {
113    var c = {
114      store: {
115        put: function () {}
116      }
117    }
118
119    try {
120      service = cache(c, service)
121      assert.fail('should not get here')
122    } catch (e) {
123      assert.equal(e.message, 'the store must implement both get and put functions')
124    }
125                                                                                                                                                                          
126  })
```
Error: Use JavaScript Standard Style (https://github.com/feross/standard)
  ~/opensource/oxr/test/test.js:112:80: Block must not be padded by blank lines
```

**Comment by nzakas** [comment]
Working on this.
